jerry-debugger/jerry_client_main.py

```diff
@@ -208,20 +212,20 @@ def __init__(self, is_func, byte_code_cp, source, source_name, line, column, nam
 
         for i, _line in enumerate(lines):
             offset = offsets[i]
-            breakpoint = JerryBreakpoint(_line, offset, self)
-            self.lines[_line] = breakpoint
-            self.offsets[offset] = breakpoint
+            br_point = JerryBreakpoint(_line, offset, self)
```

Comment:
Why do we need to rename the "breakpoint" variable? This also applies to other places.

Reply:
Rename is needed because breakopoint is a built-in.

************* Module jerry_client_main
jerry-debugger/jerry_client_main.py:215:12: W0622: Redefining built-in 'breakpoint' (redefined-builtin)